web/templates/right_sidebar.hbs

```diff
@@ -18,7 +18,7 @@
                 </button>
             </div>
             <div id="buddy_list_wrapper" class="scrolling_list" data-simplebar>
-                <ul id="user_presences" class="filters required-text" data-empty="{{t 'No matching users.' }}"></ul>
+                <ul id="narrow_user_presences" class="filters required-text" data-empty="{{t 'No matching users.' }}"></ul>
```

Comment:
For recent conversations, they actually end up in `other-users-presence-container` because there are no users that "match" the narrow, but then I also removed the two headers so the user can't tell the difference.